index.js

```diff
@@ -460,7 +460,24 @@ NYC.prototype.checkCoverage = function (thresholds) {
 }
 
 NYC.prototype._loadProcessInfoTree = function () {
-  return ProcessInfo.buildProcessTree(this._loadProcessInfos())
+  var _this = this
+  var processTree = ProcessInfo.buildProcessTree(this._loadProcessInfos())
+
+  processTree.getCoverageMap(function (filenames, maps) {
+    var map = libCoverage.createCoverageMap({})
+
+    _this._loadReports(filenames).forEach(function (report) {
```

Comment:
we can rename `_loadReports` to `loadReports`, and `process.js` can call this method on `index.js`.